winui/Shadow/Overview.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: What's New for .NET WinUI Shadow Control | Syncfusion
+description: The .NET WinUI Shadow control is used to apply shadow effects to any framework element to create a beautiful and appealing UI.
+platform:winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# WinUI Shadow (New)
+
+The [WinUI Shadow](https://www.syncfusion.com/winui-controls/shadow) .NET WinUI Shadow control is used to apply shadow effects to framework elements to give them an appearance of depth, allowing users to easily differentiate overlapping elements and enjoy a beautiful and appealing user interface.
+
+## Shadow control
```

Comment:
@Priyanka-Vijayasankar Remove the header ## Shadow control

Reply:
Removed the header.